README.md

```diff
@@ -582,6 +582,23 @@ And write to it like this:
 }
 ```
 
+### Helping with auto-discovery
+
+If you have an API consumer entity querying your Scimitar-based SCIM API provider endpoint and want to enable a degree of auto-discovery for that entity, then depending on your implementation, there may be customisations you wish to make.
+
+#### Default resources
+
+By default, Scimitar advertises (via things like [the `/Schemas` endpoint](https://tools.ietf.org/html/rfc7644#section-4)) support for both a `User` and `Group` resource, but if you (say) only support a `User` concept, you override the default using code such as this in your `config/initializers/scimitar.rb` file:
+
+```ruby
+Rails.application.config.to_prepare do
+  Scimitar::Engine::set_default_resources([Scimitar::Resources::User])
```

Comment:
@pond although this appears to work, there are linter complaints about writing it the more idiomatic way:

``` ruby
  Scimitar::Engine.set_default_resources([Scimitar::Resources::User])
```

Reply:
Yeah, fair. Thanks. I'll do a PR in the near-ish future that's just a style change for the above onto `main`/`v1` but won't release a new gem version given there's no functional change.